src/ocean/LibFeatures.d

```diff
@@ -0,0 +1,24 @@
+/*******************************************************************************
+
+    Meta module defining constants that match ocean feature releases allowing
+    downstream libraries to conditionally implement functionality relying on
+    later ocean versions.
+
+    Copyright:
+        Copyright (c) 2009-2016 Sociomantic Labs GmbH.
+        All rights reserved.
+
+    License:
+        Boost Software License Version 1.0. See LICENSE_BOOST.txt for details.
+        Alternatively, this file may be distributed under the terms of the Tango
+        3-Clause BSD License (see LICENSE_BSD.txt for details).
+
+*******************************************************************************/
+
+module ocean.LibFeatures;
+
+const has_features_2_5 = true;
+const has_features_2_4 = true;
```

Comment:
I still don't get this argument:

> However application using that library will use 2.5 at least

So why include feature constants for pre-v2.5 then? This is what I can't understand.



> Alternative would be to check for version 2.5 in that library despite the fact needed feature actually came in 2.4 - I don't mind that too much, but such situation seems much more confusing to me when someone will read that library code.

Yes, this is what I mean, for me this is less misleading, because in v2.4 there will be no feature constant so anyone using v2.4 will not get features from it used (simply because the feature constant will not be defined). So effectively features from v2.4 will only be used starting from v2.5 and on.